docs/design_decisions/DR-005-infra.rst

```diff
@@ -0,0 +1,155 @@
+..
+   Copyright (c) 2025 Contributors to the Eclipse Foundation
+
+   See the NOTICE file(s) distributed with this work for additional
+   information regarding copyright ownership.
+
+   This program and the accompanying materials are made available under the
+   terms of the Apache License Version 2.0 which is available at
+   https://www.apache.org/licenses/LICENSE-2.0
+
+   SPDX-License-Identifier: Apache-2.0
+
+DR-005-Infra: Hosting Strategy for Module Documentation
+=======================================================
+
+.. dec_rec:: Hosting strategy for module webspaces
+   :id: dec_rec__infra__webspace
+   :status: accepted
+   :context: Infrastructure
+   :decision: Combine Website from multiple Repos
+
+Context / Problem
+-----------------
+
+We host site content for modules using GitHub Pages (per-repo webspaces).
+While GitHub *repos* can store up to 10GiB (size of ``.git`` folder),
+GitHub *Pages* only allows 1GiB of storage.
+While this works so far, the storage and site limits are becoming noticeable.
+More diagrams, pull-requests, and versions will exacerbate the problem.
+
+.. plantuml::
+
+   node "Module Repo" as ModuleRepo
+   node ".gh_pages branch" as gh_pages
+   cloud "GitHub Pages" as GitHubPages
+
+   ModuleRepo -> gh_pages : "build"
+   gh_pages -> GitHubPages : "publish"
+
+In addition to GitHub Pages, the project also has the website
+https://eclipse.dev/score/ hosted via the Eclipse Foundation.
+The website is relatively small and there are plans to move most content elsewhere
+(where it can be managed via Wordpress).
+
+The foundation updates the content every 5 minutes from
+the `eclipse-score-website-published <https://github.com/eclipse-score/eclipse-score-website-published>`_ repo.
+The sources are in the `eclipse-score-website <https://github.com/eclipse-score/eclipse-score-website>`_ repo
+and get deployed automatically using the GitHub Action
+`Build and Archive <https://github.com/eclipse-score/eclipse-score-website/actions/workflows/build_and_publish.yml>`_.
+
+.. plantuml::
+
+   node "eclipse-score-website-published" as PublishedRepo
+   node "eclipse-score-website" as WebsiteSrc
+   cloud "eclipse.dev/score" as EclipseSite
+
+   WebsiteSrc -> PublishedRepo : "Build & Archive"
+   PublishedRepo <. EclipseSite : "pull every 5min"
+
+No limits for this hosting are known.
+
+Goals and Requirements
+^^^^^^^^^^^^^^^^^^^^^^
+
+1. Provide sufficient storage and bandwidth for released documentation.
+2. Keep a consistent site/URL structure and navigation across modules.
+3. Keep hosting and maintenance costs predictable and low.
+4. Dedicated space for releases?
+
+Options Considered
+------------------
+
+Option G: Continue with per-repo GitHub Pages (status quo)
```

Comment:
Can we stick to options A, B, C or use more than one letter? G, P... it's confusing for me ;)